akka-http-core/src/main/scala/akka/http/impl/engine/http2/Http2Multiplexer.scala

```diff
@@ -4,8 +4,6 @@
 
 package akka.http.impl.engine.http2
 
-import akka.http.ccompat._
```

Comment:
All the ccompat ones are actually needed but only for Scala 2.12.

Reply:
I think cats and any other library that uses implicits heavily has a lot of issues with ignoring unused imports.